### ISSUE TITLE
splash_screen_view

### ISSUE THREAD

**Issue by kasper2014** [SandipVKalola/splash_screen#4]
Hi,

is it possible to set the position for the text?
Currently it displays too low for me, and I want to raise it?

greetings

**Comment by Suzefa** [comment]
Yes you can modified the position. by just do this step if you know dart very well.....

Step 1..
SplashScreenView(
          navigateRoute: const LoginScreen(),
          duration: 3000,
          imageSrc: "your image source",
          text: "Yours Label",
          textType: TextType.TyperAnimatedText,
          textStyle: const TextStyle(
              fontWeight: FontWeight.bold,
              fontSize: 30
          ),
          backgroundColor: Colors.white,
        ),
now Step 2 press the CTRL button + click on the Widget SplashScreenView to jump on it class now if you want to modify your splash screen you can modify it there. but remember the class if from the package so edit it carefully..
hope this will help you.
